### PR TITLE
input-validation fix and contents

### PR DESCRIPTION
**Pull request by emnlpstr**
fixed input validation on form and fixed content on "soluzione-standard" page

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [x] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

### CODE REVIEW

**Review by lollomag — APPROVED** [review]

File: src/components/modal/ModalUpdates.jsx [hunk 1]
```diff
@@ -515,7 +515,7 @@ export const ModalUpdates = () => {
                           message: requiredLabel,
                         },
                         pattern: {
-                          value: /^[a-zA-Z ]*$/i,
+                          value: /^[A-zÀ-ú ]*$/i,
```
Comment (sebbalex) [inline src/components/modal/ModalUpdates.jsx:518]:
```suggestion
                          value: /^\p{L}/u,
```

File: src/components/modal/ModalUpdates.jsx [hunk 2]
```diff
@@ -552,7 +552,7 @@ export const ModalUpdates = () => {
                         message: requiredLabel,
                       },
                       pattern: {
-                        value: /^[a-zA-Zàèéìòù ]*$/i,
+                        value: /^[A-zÀ-ú_.\- ]*$/i,
```
Comment (sebbalex) [inline src/components/modal/ModalUpdates.jsx:555]:
```suggestion
                        value: /^\p{L}/u,
```

**Review by sebbalex — CHANGES_REQUESTED** [review]
I'd change the validation rules behind `enteType` and `ente` fields.
Using `/^\p{L}/u` would match any unicode Letter avoiding complexity.
More info [here](https://unicode.org/reports/tr18/#General_Category_Property)

File: src/components/modal/ModalUpdates.jsx [hunk 3]
```diff
@@ -515,7 +515,7 @@ export const ModalUpdates = () => {
                           message: requiredLabel,
                         },
                         pattern: {
-                          value: /^[a-zA-Z ]*$/i,
+                          value: /\p{L}/u,
```
Comment (sebbalex) [inline src/components/modal/ModalUpdates.jsx:518]:
```suggestion
                          value: /^\p{L}/u,
```

**Review by sebbalex — APPROVED** [review]

**Pull request merged**